docs/modules/phantompeakqualtools.md

```diff
@@ -0,0 +1,8 @@
+---
+Name: phantompeakqualtools
+URL: https://www.encodeproject.org/software/phantompeakqualtools/
+Description: >
+
```

Comment:
Please remove this whitespace and make the description a lot shorter - it has to fit into a small table on the website. Maybe:

> Computes enrichment and quality measures for ChIP-seq/DNase-seq/FAIRE-seq/MNase-seq data